lib/ticks-to-tree.js

```diff
@@ -28,23 +29,16 @@ function ticksToTree (ticks, options = {}) {
 
   // Spawn a file that throws an error to get the loading stack.
   // Then create a regular expression that matches all those files.
-  let childStderr = spawnSync(pathToNodeBinary, ['--experimental-modules', join(__dirname, 'loading-stacks')]).stderr.toString()
-  if (childStderr.includes('--experimental-modules')) {
+  const stackArgs = ['--experimental-modules', '--no-warnings', join(__dirname, 'loading-stacks')]
+  let stackChild = spawnSync(pathToNodeBinary, stackArgs)
+  if (stackChild.stderr.toString().includes('--experimental-modules')) {
     // Future proof to make sure it works even if the `experimental-modules` flag is removed.
-    childStderr = spawnSync(pathToNodeBinary, [join(__dirname, 'loading-stacks')])
+    stackArgs.shift()
+    stackChild = spawnSync(pathToNodeBinary, stackArgs)
   }
-  const stacks = childStderr
-    .match(/ \(.+?:/g)
-    .map((e) => e.slice(2))
-    .filter((e) => e[0] !== '/' && e[0] !== '\\')
-  let regExpStr = '('
-  let separator = ''
-  for (const frame of new Set(stacks)) {
-    regExpStr += `${separator} ${frame.replace(/\\/g, '\\\\').replace(/\./g, '\\.')}\\d+:\\d+`
-    separator = '|'
-  }
-  regExpStr += ')$'
-  const regExp = new RegExp(regExpStr)
+  const stackRegExpStr = stackChild.stdout.toString()
+  assert(stackRegExpStr)
```

Comment:
@BridgeAR This assertion fails for me (and CI) on Node 8 in the flame-private tests. Do you have an idea why that might be?

Reply:
Relevant flame-private issue: https://github.com/nearform/node-clinic-flame-private/issues/19